### ISSUE TITLE
Gundam Zeonic Front box art doesn't show up.

### ISSUE THREAD

**Issue by rbohannon89** [jpd002/Play-#841]
This emulator doesn't recognize that this game exists at all. You can try to reinstall the app you can try rescan library doesn't work. Since it doesn't show up it unplayable. Please fix when you can thanks.

**Comment by jpd002** [comment]
Can you give us some more details?
- What is the full file name?
- Is it the JP or US version?
- What is the disc image format? (iso, cso, isz, etc.)

Thanks!

**Comment by rbohannon89** [comment]
Yes absolutely i can. Its Mobile Suit Gundam - Zeonic Front (USA) Iso. Its
a 1.94 GB. About all know beside made by Bandai.

**Comment by jpd002** [comment]
Just to make sure, is there a dot (.) in the filename? The emulator checks for “.iso” at the end of the filename to add it in the game list.

Thanks!

**Comment by rbohannon89** [comment]
I belive this is right looks just like the others. I included screenshot
detail about the file that i can see on the device im using. If I
understood you this should work shouldn't it?

**Comment by rbohannon89** [comment]
I tried renaming the iso and take out the (USA) same result:(.

**Comment by rbohannon89** [comment]
![1578367204402_Screenshot_20200105-231039_My Files](https://user-images.githubusercontent.com/28317298/71866015-d13e2e00-30d2-11ea-89c2-370a09f7d738.jpg)



Thats the screen shot you requested.
